### DISCUSSION TITLE
Practica Html y Css

### DISCUSSION THREAD

**Discussion by R3D4N** [Launch-X-Latam/MisionFrontEnd#349] [Show and tell]
Que tal chic@s, les queria compartir mi avance de las practicas de html y css, las cuales seguire mejorando :smirk::
-Practica Html: [LaunchCake](https://r3d4n.github.io/LaunchX-Pasteleria/)

-Practica CSS: [VacunX](https://r3d4n.github.io/LaunchX-Vaccination/)

**Comment by FelipeBENIITEZ** [comment]
amigooo me gusto mucho, bien hecho

**Comment by Lu-fer-alvz** [comment]
¡¡Me encantó!! Me dieron ganas de un pastel, felicidades. :)
